### PR TITLE
Updating professional doc on headless authentication

### PR DESCRIPTION
**Pull request by stephanwampouille**
Please do not merge until @lzimm has reviewed the material

### CODE REVIEW

File: docs/operator/basic/professional.md [hunk 1]
```diff
@@ -50,6 +50,20 @@ curl -X GET -H 'Content-Type: application/json' "https://portal.prelude.org/chai
...
 
+### Managing tokens
+
+---
+
+Operator also supports the use of personal tokens that can be created and linked to your account. This pairing allows you to authenticate your log in headlessly, thereby removing the need to authenticate your account upon every log in attempt.
+
+#### Create your token and secret key
+
+To create your keys, log into your [Headquarters](https://login.prelude.org/#/) and navigate to your user Settings. Once there you will find the 'Manage Tokens' option at the bottom of the page. Here you can create and manage your tokens which can be used to authenticate your headless login:
```
Comment (privateducky) [inline docs/operator/basic/professional.md:61]: 1. is this just a professional feature?

2. how does this work or is in conflict with the above directions on using the chains API?

Comment (stephanwampouille) [reply docs/operator/basic/professional.md:61]: I'll have to run Q1 by Lewis, but from my understanding for Q2 this is just another way we support logging in. I will also have to ask Lewis if you can run the above commands using the Token you can now generate

Comment (privateducky) [reply docs/operator/basic/professional.md:61]: hint: it's not just a pro feature, it's for everyone... you should also be able to use your token to login via the steps above - my question is: does it work?

**Comment by stephanwampouille** [comment]
This one is still in progress @privateducky it got a little complicated when @lzimm and I got when I got into HMACs. We coordinated that in NY we will sit down and Lewis will help explain it to me so that I can test and in turn write about this second way of headless auth

**Pull request closed without merging**